packages/turbo-repository/rust/src/lib.rs

```diff
@@ -63,4 +98,21 @@ impl Workspace {
     pub async fn find_packages(&self) -> std::result::Result<Vec<Package>, napi::Error> {
         self.packages_internal().await.map_err(|e| e.into())
     }
+
+    #[napi]
+    pub async fn package_graph(&self) -> Result<HashMap<String, Vec<String>>, Error> {
+        let mut map = HashMap::new();
+        let packages = self.find_packages().await?;
+        let workspace_path = AbsoluteSystemPath::new(self.absolute_path.as_str()).unwrap();
```

Comment:
Should handle this unwrap as well.

Reply:
How would you handle this? This is probably a panic situation or at least need to return some error. If we don't have the workspace absolute path , we can't do the rest

Reply:
checkout 10788eab14c9c6cc41e3c4950812e2212899fe58